lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -188,15 +180,20 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
   defp has_nearby_redundancy?(%Alert{
          informed_entities: [%{facility: %{id: informed_facility_id}} | _]
        }) do
-    if data = @elevator_redundancy_data[informed_facility_id] do
-      data["nearby_redundancy?"]
-    else
-      _ =
-        Sentry.capture_message(
-          "Elevator #{informed_facility_id} does not exist in redundancy data"
-        )
+    case @elevator.get(informed_facility_id) do
+      %Elevator{redundancy: :nearby} ->
+        true
+
+      %Elevator{} ->
+        false
+
+      nil ->
+        _ =
+          Sentry.capture_message(
+            "Elevator #{informed_facility_id} does not exist in redundancy data"
+          )
```

Comment:
Should we be logging to Splunk here as well or is this a case where only want to log to Sentry?

Reply:
A previous discussion with Christian about this exact line is actually what prompted #2341. Once that's in, this will use `Screens.Log`. I think this will generally be the preferred way to log warnings/errors, since in development we'll see the printout in the console, and in deployed environments we'll get the Sentry report.

Reply:
ah, makes sense